src/streamlink/plugins/youtube.py

```diff
@@ -349,7 +358,12 @@ def _get_streams(self):
         if not is_live:
             streams, protected = self._create_adaptive_streams(info, streams, protected)
 
-        hls_playlist = info.get("hlsvp")
+        try:
+            hls_playlist = info.get("player_response").get("streamingData").get("hlsManifestUrl")
```

Comment:
you don't need  `try ... except`

use

`info.get("player_response", {}).get("streamingData", {}).get("hlsManifestUrl")`

maybe a bit ugly with so many **get**, but enough for a workaround